.scripts/autocompleteCollections.js

```diff
@@ -3,9 +3,12 @@
 // that have been applied to articles in the girder database.
 // The disease classifications are used by the autocomplete on the search page.
 
-//Leaving the localhost reference to make life easier for devs working locally
-//var mongodb = new Mongo('localhost:27017');
-var mongodb = new Mongo('172.30.2.123:27017');
+if (process.env.MONGO_URL) {
+  var mongodb = new Mongo(process.env.MONGO_URL);
```

Comment:
I don't know if the mongo library will accept the database path present in MONGO_URL.